CSSC.csv

```diff
@@ -1,3 +1,3 @@
-name, description, start_date, start_time, end_date, end_time
-CSSC's Games & Tips for First Years, A series of games and fun puzzles for our first years!, 2020-09-10, 19:00, 2020-09-10, 20:00
-Test Event, This is a test event, 2020-9-10, 13:00, 2020-10-17, 20:00
+club_name, name, description, start_date, start_time, end_date, end_time
```

Comment:
I was thinking this would be the name of the event rather than the club since the CSV would have the name of the club

Reply:
This will make the script very simple and even faster.